packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsSynchronizationStatus.tsx

```diff
@@ -1,27 +1,46 @@
+import { MessageChannelSyncStatus } from '@/accounts/types/MessageChannel';
 import { Status } from '@/ui/display/status/components/Status';
 
 export type SettingsAccountsSynchronizationStatusProps = {
-  syncStatus: 'synced' | 'failed' | 'notSynced';
+  syncStatus: MessageChannelSyncStatus;
+};
+
+const getColor = (syncStatus?: MessageChannelSyncStatus | null) => {
```

Comment:
I've just noticed we defined colors and label in message-channel object metadata already. Maybe we want to get theme from here?

```
  let objectMetadataItem = useRecoilValue(
    objectMetadataItemFamilySelector({
      objectName: CoreObjectNameSingular.MessageChannel
      objectNameType: 'singular',
    }),
  );
  ```
  Then get object from field and...

```
  const selectedOptions = field.metadata.options.filter(
    (option) => fieldValues?.includes(option.value),
  );
 ```
 
 What do you think @lucasbordeau? Would you recommend this approach to access the option values?

Reply:
Yes, we should use the options here. you can get the objectMetadataItem using useObjectMetadataItem hook directly

Reply:
@FelixMalfait,
I see options are present here - 
<img width="1199" alt="image" src="https://github.com/twentyhq/twenty/assets/60139930/6aaa69a9-867d-4ae8-96d5-7346cfd7fbb3">